weeks/week07/week07.md

```diff
@@ -1,3 +1,103 @@
 # Week 7: Web Automation with Playwright
 
-Coming soon.
+## Summary:
+Playwright is the powerful tool for Web Test automation (UI and API) in different programming languages:
+- TypeScript, 
+- JavaScript,
+- .NET,
+- Java.
+- Python
+
+It is 
```

Comment:
Better to have it as a phrase instead of a list of items. Use font formatting to highlight things like **bold**, _italic_, `i don't know how to call it`.